src/test/java/io/lettuce/core/commands/ListCommandIntegrationTests.java

```diff
@@ -267,4 +270,30 @@ void rpushxVariadic() {
         assertThat((long) redis.rpushx(key, "two", "three")).isEqualTo(3);
         assertThat(redis.lrange(key, 0, -1)).isEqualTo(list("one", "two", "three"));
     }
+
+    @Test
```

Comment:
We can also add @EnabledOnCommand("LMOVE") here since this only pass in `redis 6.2.0`. [Here](https://github.com/lettuce-io/lettuce-core/blob/5cc285adee364d39ac31b14370f047c9a13e3b94/src/test/java/io/lettuce/core/commands/SortedSetCommandIntegrationTests.java#L803)'s a similar approach.

Reply:
TIL it. Thanks!